src/store/store.development.js

```diff
@@ -21,7 +22,8 @@ export default function configureStore (rootReducer, initialState) {
       fetch,
       multi,
       promise,
-      logger
+      logger,
+      thunkMiddleware
```

Comment:
This should go before the logger but after the promise middleware in both stores.